docs/source/python/data.rst

```diff
@@ -560,3 +560,59 @@ schema without having to get any of the batches.::
    x: int64
 
 It can also be sent between languages using the :ref:`C stream interface <c-stream-interface>`.
+
+Conversion of RecordBatch do Tensor
+-----------------------------------
+
+Each array of the ``RecordBatch`` has it's own contiguous memory that is not necessarily
+adjacent to other arrays. A different memory structure that is used in machine learning
+libraries is a two dimensional array (also called a 2-dim tensor or a matrix) which takes
+only one contiguous block of memory.
+
+For this reason there is a function ``pyarrow.RecordBatch.to_tensor()`` available
+to efficiently convert tabular columnar data into a matrix.
+
+Data types supported in this conversion are unsigned, signed integer and float
+types of all widths. Currently only column-major conversion is supported.
+
+   >>>  import pyarrow as pa
+   >>>  arr1 = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+   >>>  arr2 = [10, 20, 30, 40, 50, 60, 70, 80, 90]
```

Comment:
```suggestion
   >>>  arr1 = [1, 2, 3, 4, 5]
   >>>  arr2 = [10, 20, 30, 40, 50]
```

Just to keep the vertical screen estate taken by the output a bit shorter, and it is just as illustrative as the longer version I think